File: Práctica 2/Node.java
public class Node {
    private Object data;
    private Node next;

    public Node(){
        this.data = null;
        this.next = null;
    }

    public Node(Object e){
        this.data = e;
        this.next = null;
    }

    public Object getData() {
        return data;
    }

    public Node getNext(){
        return next;
    }

    public void setData(Object data) {
        this.data = data;
    }

    public void setNext(Node next) {
        this.next = next;
    }
}

File: Práctica 3/Estructuras/Node.py
# Clase que representa un nodo de un arbol binario
class Node:
    def __init__(self, data=None):
        self._data = data
        self._left = None
        self._right = None

    def getLeft(self):
        return self._left

    def getRight(self):
        return self._right

    def getData(self):
        return self._data

    def setLeft(self, node):
        self._left = node

    def setRight(self, node):
        self._right = node

    def setData(self, data):
        self._data = data
